org.lflang/src/org/lflang/ASTUtils.java

```diff
@@ -1747,16 +1748,73 @@ public static String findAnnotationInComments(EObject object, String key) {
         return null;
     }
 
+    /**
+     * Return the value of the {@code @label} attribute if
+     * present, otherwise return null.
+     *
+     * @throws IllegalArgumentException If the node cannot have attributes
+     */
+    public static String findLabelAttribute(EObject node) {
```

Comment:
Would it make sense if this was in a separate class like `AttributeUtils`? Same for other stuff here that relate to attributes.

Reply:
Yes, I like this idea.